lib/executor.js

```diff
@@ -116,7 +116,9 @@ function setPort(app, instructions) {
     process.env.npm_package_config_port,
     app.get('port'),
     3000
-  ], _.isFinite);
+  ], function(v) {
+    return _.isFinite(parseInt(v, 10));
+  });
```

Comment:
> I think the following may serve pretty well:

re: your function, it does look like a good replacement for the lodash function.  I'll update the patch to use that and try a version with my suggestion from above.